<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>removerow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>make_windows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>removerow()</slot>
  <slot>make_windows()</slot>
  <slot>search()</slot>
 </slots>
</ui>
